index.html

```diff
@@ -15,6 +15,98 @@
 
     <!-- Add your markup here -->
 
+  
+  <main>
+  
+  <header>
+    <nav class="navbar container">
+      <img class="logo" src="https://image.shutterstock.com/z/stock-vector-vector-graffiti-seamless-pattern-with-abstract-colorful-bright-tags-letters-without-meaning-1056235229.jpg" alt="Logo image" width="60"/>
+      <ul class= "navbar list">
+        <li class= "list-item"><a href="" class="nav-link">Project</a></li>
+        <li class= "list-item"><a href="" class="nav-link">About</a></li>
+        <li class= "list-item"><a href="" class="nav-link">Contact</a></li>
+      </ul>
+      <h2 class="navbar-text">Urban Adventures - Graffiti Art </h2>
+  </header>
+    </main>
+
+    
+  
+
+ 
+            
+    <section>
+        <div class="row">
+            <h2 class="Project">Project</h2>
+
+            <article class="col-4">
+                <img class="col-12" src="https://lainie10.files.wordpress.com/2013/10/graffiti.jpg"  />
+            </article>
+            <article class="col-4">
+                <img class="col-12" src="https://wallup.net/wp-content/uploads/2016/02/206515-walls-graffiti-colorful-Miami-nature.jpg"  />
```

Comment:
Don't use `col-...` classes on `img` tags, it clashes with the sizing of the image. 
Remove it, and set your images to `width: 100%` instead